src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -5,6 +5,7 @@
 using Azure.Data.AppConfiguration;
 using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
 using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.JsonConfiguration;
```

Comment:
>JsonConfiguration [](start = 63, length = 17)

I'm not sure if this new namespace is valuable. Would it be fine if we just put the types in the `Microsoft.Extensions.Configuration.AzureAppConfiguration` namespace?